test/test_quantized_models.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.jit
+from common_utils import run_tests
+from common_quantization import QuantizationTestCase, ModelMultipleOps
+
+class ModelNumerics(QuantizationTestCase):
+    def test_float_quant_compare(self):
+        torch.manual_seed(42)
+        myModel = ModelMultipleOps().to(torch.float32)
+        myModel.eval()
+        calib_data = torch.rand(1024, 3, 15, 15, dtype=torch.float32)
+        eval_data = torch.rand(1, 3, 15, 15, dtype=torch.float32)
+        out_ref = myModel(eval_data)
+        qModel = torch.quantization.QuantWrapper(myModel)
+        qModel.eval()
+        qModel.qconfig = torch.quantization.default_qconfig
+        torch.quantization.fuse_modules(qModel.module, [['conv1', 'bn1', 'relu1']])
+        torch.quantization.prepare(qModel)
+        qModel(calib_data)
+        torch.quantization.convert(qModel)
+        out_q = qModel(eval_data)
+        SQNR = 20 * torch.log10(torch.norm(out_ref) / torch.norm(out_ref - out_q))
+        self.assertGreater(SQNR, 30, msg='Quantized model numerics diverge from float, expect SQNR > 30 dB')
```

Comment:
How did you decide on 30?

Reply:
I want a value that is as high as possible. With 8 bits of precision the maximum we could get is around 40 dB (~5 dB per bit). With a cascade of many layers, what is achievable is below this. I see 35-40 dB when I run it for different random seeds. The goal is to have a tight check so that we can find any numerics errors.

Reply:
tight check might result in flaky test depending on what random values are generated.

Reply:
There is no randomness in the test. I fix the random seed so that the numerics do not change.